### ISSUE TITLE
Search in Favorite description 

### ISSUE THREAD

**Issue by alele-de** [osmandapp/OsmAnd#19077]
### Describe the idea (required)

Hi OsmAnd Team,

Is it possible to implement a fulltext search for the description of favorites?
If you for example plan an Journey and set interesting favorites you want to visit with appropriate keywords inside, it would be perfect to find it. 

Thank you and kind regards,
Alex

### Tell us about the expected behaviour (required)

See idea 

### Tell us about alternatives you've considered (required)

No workarounds

**Comment by yuriiurshuliak** [comment]
Yes you're right. Currently, the search by description does not work, only the search by the name of the favorite.

OsmAnd~ 4.7.0#1578m, released: 2024-02-12

https://github.com/osmandapp/OsmAnd/assets/127092082/6d4df84d-7bd6-4158-8b73-1d7b10f53305

**Comment by alele-de** [comment]
> Yes you're right. Currently, the search by description does not work, only the search by the name of the favorite.
> 
> 
> 
> OsmAnd~ 4.7.0#1578m, released: 2024-02-12
> 
> 
> 
> https://github.com/osmandapp/OsmAnd/assets/127092082/6d4df84d-7bd6-4158-8b73-1d7b10f53305
> 
> 
> 
> 
> 
> 

Hi! When do you think will release 4.7 released in the apple store?
KR, Alex
